backend/src/backend/__init__.py

```diff
@@ -0,0 +1,5 @@
+from . import managers
+
+
+def hello() -> str:
+    return "Hello from backend!"
```

Comment:
🗑️ 🚮

Reply:
Oopsies, fixed in 67279b52f5197d304496d556bdbec96deea1d847